src/vellum/workflows/nodes/displayable/code_execution_node/utils.py

```diff
@@ -74,8 +75,25 @@ def run_code_inline(
 ) -> Tuple[str, Any]:
     log_buffer = io.StringIO()
 
+    VELLUM_TYPES = get_args(VellumValue)
+
+    def wrap_value(value):
+        if isinstance(value, list):
+            return ListWrapper(
+                [
+                    # Convert VellumValue to dict with its fields
+                    (
+                        item.model_dump()
+                        if isinstance(item, VELLUM_TYPES)
```

Comment:
[nit] I _think_ we could avoid the nested method by just adding `elif isinstance(item, VELLUM_TYPES)` as a case within `_clean_for_dict_wrapper`